runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -47,7 +47,6 @@ class TR_J9ServerVM: public TR_J9VM
 
    virtual bool storeOffsetToArgumentsInVirtualIndirectThunks() override { return true; }
    virtual bool needsContiguousCodeAndDataCacheAllocation() override     { return true; }
-   virtual bool supportsEmbeddedHeapBounds() override                    { return false; }
```

Comment:
`TR_J9SharedCacheServerVM` should continue to return false much like the `TR_J9SharedCacheVM` does

Reply:
I did not remove the override to false for TR_J9SharedCacheServerVM.

Reply:
Ok. I didn't expect the implementation for `TR_J9SharedCacheServerVM` to exist because it should have used the one from `TR_J9ServerVM` which it derives from.